mappings/net/minecraft/text/Text.mapping

```diff
@@ -62,6 +62,13 @@ CLASS net/minecraft/class_2561 net/minecraft/text/Text
 		COMMENT {@return whether the text contains {@code text}, without considering styles}
 		ARG 1 text
 	METHOD method_44746 withoutStyle ()Ljava/util/List;
+	METHOD method_48321 translatableWithFallback (Ljava/lang/String;Ljava/lang/String;)Lnet/minecraft/class_5250;
+		ARG 0 key
+		ARG 1 fallback
+	METHOD method_48322 translatableWithFallback (Ljava/lang/String;Ljava/lang/String;[Ljava/lang/Object;)Lnet/minecraft/class_5250;
```

Comment:
Could these not just be `translatable`? I feel the `fallback` argument name is enough to get that these are ones with fallback.

Reply:
This is for user convenience - to allow calling `translatable(String, ...Object)` without weird casts.

Reply:
Ah, yeah that makes sense.